canvas_modules/common-canvas/__tests__/common-properties/components/table-toolbar-test.js

```diff
@@ -851,117 +901,127 @@ describe("TableToolbar multi select edit button works correctly", () => {
-		expect(multiSelectEditButton).to.have.length(1);
+		tableToolbar = container.querySelector("div.properties-table-toolbar");
+		multiSelectEditButton = tableToolbar.querySelector("button.properties-action-multi-select-edit");
+		expect(multiSelectEditButton).to.exist;
 
-		// Click the multiSelectEditButton in table toolbar
-		multiSelectEditButton.simulate("click");
+		// Click the multiSelectEditButton in table toolbars
+		fireEvent.click(multiSelectEditButton);
 
 		// A new panel opens which shows editable columns
-		let wideFlyoutPanel = wrapper.find(".properties-wf-children");
-		let editableColumns = wideFlyoutPanel.find(".properties-editstyle-inline").find(".properties-ctrl-wrapper");
+		let wideFlyoutPanel = container.getElementsByClassName("properties-wf-children");
+		let editableColumns = [];
+		for (let i = 0; i < wideFlyoutPanel.length; i++) {
+			const col = wideFlyoutPanel[i].querySelector(".properties-ctrl-wrapper");
+			if (col.parentElement.className.includes("properties-control-panel")) {
+				editableColumns.push(col);
+			}
+		}
 		expect(editableColumns).to.have.length(2); // Animals column has edit_style: "subpanel". Can't edit from selectedEditCells.
 
 		// Set 44 for Integer field
-		let integerNumber = editableColumns.at(0).find("input");
-		integerNumber.simulate("change", { target: { value: "44" } });
-
+		let integerNumber = within(editableColumns[1]).getAllByRole("spinbutton")[0];
+		fireEvent.change(integerNumber, { target: { value: "44" } });
 
 		// Save wide flyout
-		wrapper.find(".properties-modal-buttons").find("button.properties-apply-button")
-			.at(0)
-			.simulate("click");
-
+		let buttons = container.getElementsByClassName("properties-apply-button");
+		fireEvent.click(buttons[1]);
 		// verify that the values have changed in the selected rows.
-		summaryPanel = propertyUtils.openSummaryPanel(wrapper, "SLE_mse_panel");
-		tableRows = tableUtils.getTableRows(summaryPanel);
+		summaryPanel = propertyUtilsRTL.openSummaryPanel(wrapper, "SLE_mse_panel");
+		tableRows = tableUtilsRTL.getTableRows(summaryPanel);
 		tableRows.forEach((row) => {
-			const integerField = row.find("input").at(1);
-			expect(integerField.props()).to.have.property("value", 44);
+			const integerField = within(row).getAllByRole("spinbutton")[0];
+			expect(integerField.value).to.equal("44");
 		});
 
 		// Click the multiSelectEditButton in table toolbar
-		multiSelectEditButton.simulate("click");
+		fireEvent.click(multiSelectEditButton);
 
 		// A new panel opens which shows editable columns
-		wideFlyoutPanel = wrapper.find(".properties-wf-children");
-		editableColumns = wideFlyoutPanel.find(".properties-editstyle-inline").find(".properties-ctrl-wrapper");
+		wideFlyoutPanel = container.getElementsByClassName("properties-wf-children");
+		editableColumns = [];
+		for (let i = 0; i < wideFlyoutPanel.length; i++) {
+			const col = wideFlyoutPanel[i].querySelector(".properties-ctrl-wrapper");
```

Comment:
here can you also check within `properties-editstyle-inline` class ?

Reply:
For some reason, `properties-editstyle-inline` doesn't exist. It isn't there in the html, and the querySelector can't locate it.